lib/xlsx/xform/drawing/two-cell-anchor-xform.js

```diff
@@ -106,14 +106,16 @@ utils.inherits(TwoCellAnchorXform, BaseXform, {
   },
 
   reconcile: function(model, options) {
-    var rel = options.rels[model.picture.rId];
-    var match = rel.Target.match(/.*\/media\/(.+[.][a-z]{3,4})/);
-    if (match) {
-      var name = match[1];
-      var mediaId = options.mediaIndex[name];
-      model.medium = options.media[mediaId];
+    if (model.picture && model.picture.rId) {
+      var rel = options.rels[model.picture.rId];
+      var match = rel.Target.match(/.*\/media\/(.+[.][a-z]{3,4})/);
+      if (match) {
+        var name = match[1];
+        var mediaId = options.mediaIndex[name];
+        model.medium = options.media[mediaId];
+      }
     }
-    if (Number.isInteger(model.tl.row) && Number.isInteger(model.tl.col) && Number.isInteger(model.br.row) && Number.isInteger(model.br.col)) {
+    if (model.tl && Number.isInteger(model.tl.row) && Number.isInteger(model.tl.col) && Number.isInteger(model.br.row) && Number.isInteger(model.br.col)) {
```

Comment:
Is it worth adding a test for `model.br` too? Obviously I don't know if it's possible for it to be null or not, but the code doesn't give any indication either way.

Reply:
Great point. I went ahead and added a `br` guard and test. I should have done that originally.

Reply:
I started with the `tl` guard because I was testing with `model = {}`, which threw an error on that conditional. I didn't have the time to debug everything enough to figure out where `model` was coming from and why its `picture` was `null`, so I'm as in the dark about whether or not `br` can be `null` without a `null` `tl` as you are. It doesn't hurt, so there's no reason not to.